Content.Client/MassMedia/Ui/NewsReadMenu.xaml

```diff
@@ -0,0 +1,52 @@
+<DefaultWindow
+    xmlns="https://spacestation14.io"
+    xmlns:gfx="clr-namespace:Robust.Client.Graphics;assembly=Robust.Client"
+    xmlns:controls="clr-namespace:Content.Client.UserInterface.Controls"
+    Title="{Loc 'news-read-ui-default-title'}"
+    MinSize="512 512"
+    SetSize="512 512">
+    <BoxContainer Orientation="Vertical">
```

Comment:
Why not just hide this element and display "No articles found" or something when the UI gets updated with a null article?